### ISSUE TITLE
admin option on lpass share useradd doesn't work

### ISSUE THREAD

**Issue by apelliciari** [lastpass/lastpass-cli#150]
Hi,

i'm adding a group through the CLI to a shared folder

```shell
lpass share useradd --read-only=false --hidden=false --admin=true My-Shared-Folder My-Group
```
The command runs successfully, but when I look into my Vault this group was added without admin permissions.

Is this a bug?

Thanks,
Alessandro

**Comment by bcopeland** [comment]
Hmm, it may be that we support it for users but not groups; can you tell whether that's the case?

**Comment by apelliciari** [comment]
Yes, I can confirm I was adding a group. Not tried with users.

I've looked at the code but unfortunately I'm not so confident with C and I can't understand how to fix it.

**Comment by bcopeland** [comment]
Thanks for the report!  This should be fixed now by 74b0ebf572e096f557e5eb54a9d9494a8a17ece0.
